full/src/main/java/apoc/custom/CypherProcedures.java

```diff
@@ -104,7 +101,7 @@ public void asFunction(@Name("signature") String signature, @Name("statement") S
                 ProcedureSignature signature = procedureDescriptor.getSignature();
                 return new CustomProcedureInfo(
                         PROCEDURE,
-                        signature.name().name(),
+                        getFullStatement(signature.name()),
```

Comment:
prova così e rimuovi il metodo
```suggestion
                        signature.name().toString(),
```